tests/python/requirements.txt

```diff
@@ -8,10 +8,7 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-# This file enumarates the various third party libraries Toolkit uses for unit testing.
-
-# Used for code coverage
-coverage==4.4.1
```

Comment:
Coverage is not installed with the 3rd parties anymore, because there's a compiled component to it and we don't want to commit those with tk-core. It was removed in the past from the repo, but we forgot to update the requirements.txt.